AppleStore/AppleStore/DetailProductViewController.swift

```diff
@@ -7,33 +7,196 @@
 
 import UIKit
 
-final class DetailProductViewController: UIViewController {
+final class DetailProductViewController: UIViewController, UIScrollViewDelegate {
```

Comment:
```suggestion
final class DetailProductViewController: UIViewController {
```
, UIScrollViewDelegate  в экстешнен все делегаты